cabal-install/src/Distribution/Client/Errors.hs

```diff
@@ -33,6 +33,8 @@ import Distribution.Pretty
 import Distribution.Simple (VersionRange)
 import Distribution.Simple.Utils
 import Network.URI
+import Text.PrettyPrint hiding (render, (<>))
+import qualified Text.PrettyPrint as PP
```

Comment:
Why not everything qualified?
Would make easier to recognise a function/operator is from `Text.PrettyPrint` at first sight.

Reply:
I'm using the qualified import for disambiguation. There'd be a lot of `PP.` noise in the following if I didn't use the unqualified import.

![image](https://github.com/user-attachments/assets/74338b5a-9dc8-402d-a59d-464f05918c3f)